XamarinCommunityToolkitSample/Pages/WelcomePage.xaml.cs

```diff
@@ -29,5 +29,10 @@ Page GetPage(WelcomeSectionId id)
             page.Title = id.GetTitle();
             return page;
         }
+
+        async void OnAboutClicked(System.Object sender, System.EventArgs e)
+        {
```

Comment:
no need in bracers )

Reply:
You mean using the new 1 line method call notation? '=>'

Reply:
Yeah